android/src/main/java/com/reactnativeescposprinter/EscPosPrinterModule.java

```diff
@@ -312,10 +312,9 @@ private void disconnectPrinter() {
       mPrinter.clearCommandBuffer();
     }
 
-    private void printData() throws Epos2Exception {
+    private void printData(int timeout) throws Epos2Exception {
       this.connectPrinter();
-      mPrinter.sendData(Printer.PARAM_DEFAULT);
-
+      mPrinter.sendData(timeout > 5000 ? timeout : Printer.PARAM_DEFAULT);
```

Comment:
With all further comments I've passed, I suppose it'll better to use it like 
```
mPrinter.sendData(timeout || Printer.PARAM_DEFAULT);
```

Reply:
will be changed as I will switch to use a params object